process/transaction/process.go

```diff
@@ -15,14 +17,19 @@ import (
 
 var log = logger.DefaultLogger()
 
+var MinGasPrice = int64(1)
```

Comment:
Comments missing on exported members. Also, they can be constants

Reply:
done